### DISCUSSION TITLE
Assembling Control Board BTTF TIME CICRCUIT

### DISCUSSION THREAD

**Discussion by Gabin37** [CircuitSetup/Time-Circuits-Display#44] [General]
Hi, 

Got problem for the enter button, I don't know if I cut at the right place. Could you please tell me? Maybe I cut too much or is normal this button is more pressed than the other ?
![IMG_5112](https://github.com/CircuitSetup/Time-Circuits-Display/assets/154004044/987d1dfa-0f67-4d72-87be-e27bac3522e3)
![IMG_5114](https://github.com/CircuitSetup/Time-Circuits-Display/assets/154004044/a7b1dbc6-1a84-4271-9d40-0c0619e76246)

**Comment by realA10001986** [comment]
It is pushed up by the button (and the little white spacer) when the PCB is mounted.

**Comment by Gabin37** [comment]
on the pic, the PCB is mounted... so Did I cut the button too short?

**Comment by CircuitSetup** [comment]
> mounted

Is the white spacer in there? It should still be okay with the spacer, even though it's cut a bit short.

**Comment by Gabin37** [comment]
Oh ! You re right ! I ´ve forgotten to put it! Now it s perfect ! Thanks for remind me this :)GabinLe 16 déc. 2023 à 19:55, John ***@***.***> a écrit :﻿

mounted

Is the white spacer in there. It should still be okay with the spacer, even though it's cut a bit short.
